modules/uploads/config.ts

```diff
@@ -0,0 +1,16 @@
+export interface Config {
+	maxUploadSize: UploadSize;
+	maxMultipartUploadSize: UploadSize;
+	maxFilesPerUpload?: number;
+	defaultMultipartChunkSize?: UploadSize;
+}
+
+export const DEFAULT_MAX_FILES_PER_UPLOAD = 10;
+// export const DEFAULT_MULTIPART_CHUNK_SIZE: UploadSize = { mib: 100 };
+export const DEFAULT_MULTIPART_CHUNK_SIZE: UploadSize = { mib: 10 };
+
+type Units = "b" | "kb" | "mb" | "gb" | "tb" | "kib" | "mib" | "gib" | "tib";
+
+export type UploadSize = {
+	[unit in Units]: Record<unit, number>;
+}[Units];
```

Comment:
This is nice, but let's:

(a) Move it to a util file that can be shared across modules. Probably `/utils/data_size.ts`
(b) Use a string instead of an object. No need to write TypeScript types for string validation, keep it simple.

Then update the user upload to support these file formats.